vispy/scene/widgets/widget.py

```diff
@@ -44,9 +45,15 @@ class Widget(Compound):
     def __init__(self, pos=(0, 0), size=(10, 10), border_color=None,
                  border_width=1, bgcolor=None, padding=0, margin=0, **kwargs):
         # For drawing border.
-        # A mesh is required because GL lines cannot be drawn with predictable
-        # shape across all platforms.
-        self._mesh = MeshVisual(color=border_color, mode='triangles')
+        self._border = _BorderVisual(pos=pos,
```

Comment:
Yes, I agree. However, I'm not sure what the neat solution to that is..? Like, it's a code versus performance trade-off right?